docs - update saml & sso (#22954)
metabase/metabase: docs/administration-guide/10-single-sign-on.md

- Updates to a person's group membership based on LDAP mappings are not instantaneous; the changes will take effect only after people log back in.
- People are only ever added to or removed from mapped groups; the sync has no effect on groups in your Metabase that don't have an LDAP mapping.

### LDAP group membership filter

{% include plans-blockquote.html feature="LDAP advanced features" %}

Group membership lookup filter. The placeholders {dn} and {uid} will be replaced by the user's Distinguished Name and UID, respectively.

## Further reading

- [Using LDAP for authentication and access control](/learn/permissions/ldap-auth-access-control.html)
- [LDAP troubleshooting guide](../troubleshooting-guide/ldap.md)
- [Permissions overview](05-setting-permissions.md)

## Syncing user attributes at login

{% include plans-blockquote.html feature="Advanced authentication features" %}

### Syncing attributes with LDAP

> LDAP user attributes are managed from your LDAP directory. If you've set up [data-sandboxing-docs][data-sandboxing-docs], these attributes will be [automatically passed to Metabase][getting-user-attributes].

Reviewer: Can you rephrase to active voice so it's clear what's managing and passing attributes?

### Syncing attributes with Google

User attributes can't be synced with regular Google Sign-In. You'll need to set up [Google SAML][google-saml-docs] or [JWT][jwt-docs] instead.

> ## Changing a user's login method from email to SSO

Reviewer: ```suggestion
## Changing an account's login method from email to SSO
```

Unless it's awkward or incorrect to omit "user", avoid the term "user". Prefer people or account.

> You cannot use the Admin panel to update the authentication method for a user once they have set it up for the first time.

Reviewer: ```suggestion
Once a person creates an account, you cannot change the authentication method for that account.
```

Should we suggest the workaround here?

Metabase accounts created with Google Sign-In do not have passwords and must use Google to sign in to Metabase.

> For Metabase accounts created with emails and passwords, you can:
> - Deactivate password authentication for all users from **Admin settings** > **Authentication**. You'll need to ask users to sign in with Google if they haven't already.
> - Manually update the user's login method in the Metabase application database. This option is not recommended unless you are very familiar with making changes to the application database.
> 
> Note that you must have at least one user with email and password login. This ensures that you won't get locked out of Metabase if there are any problems with your SSO provider.

Reviewer: ```suggestion
For Metabase accounts created with emails and passwords, you can:
- Deactivate password authentication for all users from **Admin settings** > **Authentication**. You'll need to ask people to sign in with Google (if they haven't already).
- Manually update the account's login method in the Metabase application database. This option is not recommended unless you're  familiar with making changes to the application database.

Note that you must have at least one account with email and password login. This account safeguards you from getting locked out of your Metabase if there are any problems with your SSO provider.
```
